src/EditorFeatures/Core/Commands/GoToPreviousMethodCommandArgs.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Editor;
+
+namespace Microsoft.CodeAnalysis.Editor.Commands
+{
+    [ExcludeFromCodeCoverage]
+    internal class GoToPreviousMethodCommandArgs : CommandArgs
```

Comment:
Yeah, it's a little ambiguous.  The VS command is "Edit.NextMethod", but the implementation is (and has always been) to go to the next member.  That's why I have the split - the command args mimics the VS command, but the impl is in terms of what it actually does.